src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/PackageModel.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+#nullable enable
+
+using System;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.UI
+{
+    public abstract class PackageModel
+    {
+        protected PackageModel(PackageIdentity identity, string title, string description, string authors, Uri projectUrl, string[] tags, DateTimeOffset? published)
```

Comment:
If these parameters are truly optional, should we set defaults so the constructor can be shorter and not require passing nulls for all the optional parameters?